components/wpt-results.html

```diff
@@ -180,23 +180,6 @@
         this._refreshDisplayedSpecs()
       }
 
-      _specFilter(spec) {
-        if (this.query.length === 0) {
-          return true
-        }
-
-        const matchesQuery = (tf) => tf.testFile.toLowerCase().includes(this.query.toLowerCase())
-        return spec.testFiles.some(matchesQuery)
-      }
-
-      _testFileFilter(testFile) {
-        if (this.query.length < 3) {
-          return false
-        }
```

Comment:
This was actually unused, the logic also appears in `_refreshDisplayedSpecs`:
https://github.com/GoogleChrome/wptdashboard/blob/master/components/wpt-results.html#L249

So this deletion has no behavioral effect.